apps/dokploy/components/dashboard/settings/web-domain.tsx

```diff
@@ -78,6 +81,7 @@ export const WebDomain = () => {
 			host: data.domain,
 			letsEncryptEmail: data.letsEncryptEmail,
 			certificateType: data.certificateType,
+			wildcardDomain: data.wildcardDomain,
```

Comment:
This is not updating the value in the database, check in the backend mutation how the data is received.